src/classes/LightningMetadataController_Tests.cls

```diff
@@ -0,0 +1,34 @@
+@isTest
+private class LightningMetadataController_Tests {
+
+    @isTest
+    static void getSObjectMetadataTest() {
```

Comment:
Method `getSObjectMetadataTest` doesn't follow naming convention. Please rename it using naming convention: Lower camel case with underscores (i.e. myName_with_moreParts).